bin/prepare_backlog_study.py

```diff
@@ -27,10 +27,17 @@
 
 
 def main():
+    validation_tasks = ['aggregation_check', 'assembly_check', 'vcf_check']
+
     argparse = ArgumentParser(description='Prepare to process backlog study and validate VCFs.')
     argparse.add_argument('--eload', required=True, type=int, help='The ELOAD number for this submission')
     argparse.add_argument('--force_config', action='store_true', default=False,
                           help='Overwrite the configuration file after backing it up.')
+    argparse.add_argument('--validation_tasks', required=False, type=str, nargs='+',
+                          default=validation_tasks, choices=validation_tasks,
+                          help='task or set of tasks to perform during validation')
```

Comment:
This totally makes sense, can you just update the readme as well?

Reply:
Done